examples/RGBCalibrate/RGBCalibrate.ino

```diff
@@ -65,8 +88,7 @@ void loop() {
     leds[2] = CRGB(0,255,0);
     leds[3] = CRGB(0,0,255);
     leds[4] = CRGB(0,0,255);
-    leds[5] = CRGB(0,0,255);
-    leds[6] = CRGB(0,0,0);
```

Comment:
Noticed and fixes a latent defect here - the array is only 6 entries long, so this is a buffer overflow as-is.

Reply:
leds[5] still needs to be set to blue (0,0,255) though.

Reply:
I thought the last led should still be black as apparently was intended in the original code, but I can easily change this.

Reply:
I've made the change